hail/python/hailtop/batch/backend.py

```diff
@@ -425,9 +425,11 @@ def __init__(self,
                     'Use the remote_tmpdir parameter to specify a path.')
             remote_tmpdir = f'gs://{bucket}/batch'
         else:
-            if not remote_tmpdir.startswith('gs://'):
+            schemes = {'file', 'gs', 'hail-az', 's3'}
```

Comment:
I think Dan wants us to remove `s3` here for now.

Reply:
The RouterAsyncFS definition above needs to either have Azure added or I think it's an item on the to-do list already.